src/main/java/com/streisky/virtualshowcasebackend/config/docs/SpringDocConfigurations.java

```diff
@@ -0,0 +1,18 @@
+package com.streisky.virtualshowcasebackend.config.docs;
+
+import io.swagger.v3.oas.models.Components;
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.security.SecurityScheme;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class SpringDocConfigurations {
+
+    @Bean
+    public OpenAPI customOpenAPI() {
+        return new OpenAPI().components(new Components()
+                .addSecuritySchemes("bearer-key",
+                        new SecurityScheme().type(SecurityScheme.Type.HTTP).scheme("bearer").bearerFormat("JWT")));
```

Comment:
Formatting is a bit weird IMO.
I would recommend setting up [code style](https://www.jetbrains.com/help/idea/configuring-code-style.html) on IDE.  Searching the internet I see that you can pickup famous ones like [google's](https://github.com/google/styleguide/blob/gh-pages/intellij-java-google-style.xml), though I would still recommend you to set up a code style reviewing item by item and comparing patterns across popular ones.

On a second note, you can set up a linter like [checkstyle](https://github.com/checkstyle/checkstyle) (not a recommendation but quickly looking it up, seems like it is popular for Java). Then you could run it before each commit, run it automatically through a CI on Github Actions or just set up a [pre commit hook](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks) so it runs whenever you commit to your local branch.

```suggestion java
    public OpenAPI customOpenAPI() {
        return new OpenAPI().components(
            new Components().addSecuritySchemes(
                "bearer-key",
                new SecurityScheme().type(
                    SecurityScheme.Type.HTTP).scheme("bearer").bearerFormat("JWT")
                )
            );
```

Reply:
For now, it will be kept this way. However, will be create a PR in the future to configure to run Git Actions with checkstyle verification and run unit tests.